libs/agno/agno/agent/agent.py

```diff
@@ -211,13 +211,11 @@ class Agent:
     # Role for the system message
     system_message_role: str = "system"
     # If True, create a default system message using agent settings and use that
-    create_default_system_message: bool = True
+    process_context: bool = True
```

Comment:
```suggestion
    build_context: bool = True
```
Remember to update elsewhere